hcl2template/types.packer_config.go

```diff
@@ -362,8 +362,7 @@ func (cfg *PackerConfig) GetBuilds(opts packer.GetBuildsOptions) ([]packer.Build
 				PostProcessors: pps,
 				Prepared:       true,
 			}
-			// Prepare just sets the "prepareCalled" flag on CoreBuild, since
-			// we did all the prep here.
+			// Prepare just sets the "prepareCalled" flag on CoreBuild, since // we did all the prep here.
```

Comment:
ops, the comment lost its original format.

Reply:
Ah thanks!